.github/actions/3-build-native/action.yml

```diff
@@ -12,6 +12,24 @@ runs:
   using: composite
   steps:
 
+    - name: 'Linux x86_64: Build symgc'
+      if: runner.os == 'Linux' && inputs.arch == 'x86_64'
+      shell: bash
+      run: |
+        set -eux
+        if ! type -P dub &>/dev/null; then
+          echo "No dub from host compiler in PATH (expected on Alpine), skipping symgc build."
+          exit 0
+        fi
+        cd runtime/symgc
+        # FIXME: hitting LLVM assertion with -g
+        DFLAGS="-O -gline-tables-only -singleobj" dub build -v --compiler=$PWD/../../../bootstrap-ldc/bin/ldc2
```

Comment:
`-singleobj` compiles the whole symgc project to a single object file (in the produced static lib) - which is then LLVM-optimized as a single unit (=> implicit cross-module optimizations).